test-util/src/main/java/org/jboss/windup/testutil/html/TestJavaApplicationOverviewUtil.java

```diff
@@ -77,6 +77,7 @@ public void checkMainEffort(int expectedEffort)
                                 ".//div[contains(@class, 'container') and contains(@class, 'mainGraphContainer')]//div[@class = 'points']/div[@class = 'number']"));
         String effortString = effortElement.getText().trim();
         effortString = effortString.replace(",", "");
+        effortString = effortString.replace(".", "");
```

Comment:
It might be useful if our tests did pass in other locales, though. Maybe this would be sufficient?